tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -51,3 +51,27 @@ Feature: full text search
       | dav-path-version |
       | old              |
       | new              |
+
+
+  Scenario Outline: as a sharee search a file tagged by sharee using tag
```

Comment:
Now I think, what if we merge this scenario with above. create one tag by sharer and another by sharee. Sharee searches and then gets both files

Reply:
Yeah! It can be merged. I will do it